src/main/java/org/dominokit/domino/ui/forms/SuggestBox.java

```diff
@@ -64,15 +72,17 @@ private void search() {
         loader.start();
```

Comment:
can we move this into the null check so the loader wont start unless there is a store.

Reply:
Yep, makes sense!